config/settings/base.py

```diff
@@ -35,6 +35,7 @@
 # ------------------------------------------------------------------------------
 # Hosts/domain names that are valid for this site
 ALLOWED_HOSTS = env.list('DJANGO_ALLOWED_HOSTS', default=['*'])
+USE_X_FORWARDED_HOST = env.bool('DJANGO_USE_X_FORWARDED_HOST', default=False)
```

Comment:
Please remove the redundant `default` kwarg and just provide `False`, same with `env.str()` in `production.py`. Otherwise looking good.

Reply:
I'm afraid the 'default' is required. Without the default I can build the Docker image:

`django.core.exceptions.ImproperlyConfigured: Set the DJANGO_USE_X_FORWARDED_HOST environment variable`

Reply:
In the same settings file, there are multiple examples of `env.bool()` working with the default provided as an arg instead of a kwarg. This is just a variable like any other so it should work as well. Maybe you've made some other mistake?

Reply:
I believe that this works in the other case just due to some (?) default setting in the Docker setup for that variable.
IMHO the default setting is not redundant, as one can see from this small experiment:

```
$ unset FOO
$ python
Python 3.10.11 (main, May 15 2023, 15:04:04) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import environ
>>> env = environ.Env()
>>> print(env.bool('FOO', default=False))
False
>>> print(env.bool('FOO'))
Traceback (most recent call last):
  File "/home/wilhelmh/.virtualenvs/sodar-server/lib/python3.10/site-packages/environ/environ.py", line 403, in get_value
    value = self.ENVIRON[var_name]
  File "/usr/local/python3.10/lib/python3.10/os.py", line 680, in __getitem__
    raise KeyError(key) from None
KeyError: 'FOO'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/wilhelmh/.virtualenvs/sodar-server/lib/python3.10/site-packages/environ/environ.py", line 246, in bool
    return self.get_value(var, cast=bool, default=default)
  File "/home/wilhelmh/.virtualenvs/sodar-server/lib/python3.10/site-packages/environ/environ.py", line 407, in get_value
    raise ImproperlyConfigured(error_msg) from exc
django.core.exceptions.ImproperlyConfigured: Set the FOO environment variable
```

In case you want to drop the default in the code, we have to add it elsewhere.

Reply:
The default should of course not be dropped. It should be provided as an arg, not a kwarg.

Reply:
Sorry - completely missed your point. Will fix that.

Reply:
Have just submitted another squashed commit.

Reply:
I could have been more clear, apologies. Looking good now, I'll merge :)